lib/Target/Jaxite/JaxiteEmitter.cpp

```diff
@@ -229,6 +236,62 @@ LogicalResult JaxiteEmitter::printOperation(tensor::FromElementsOp op) {
   return success();
 }
 
+// Loading variables.
+// Example: temp_nodes[idx] = input[i]
+LogicalResult JaxiteEmitter::printOperation(memref::LoadOp op) {
+  emitAssignPrefix(op.getResult());
+  os << variableNames->getNameForValue(op.getMemref());
+  os << bracketEnclosedValues(op.getIndices(), [&](Value value) {
+    SmallString<16> idx_str;
+    dyn_cast<IntegerAttr>(
+        dyn_cast<arith::ConstantOp>(value.getDefiningOp()).getValue())
+        .getValue()
+        .toStringUnsigned(idx_str);
+    return idx_str.c_str();
```

Comment:
something strange might be going on with returning the c_str from the SmallString - you might need to just return `std::string(idx_str)`